github/github.go

```diff
@@ -481,25 +489,33 @@ func (r *Response) populatePageValues() {
 			if err != nil {
 				continue
 			}
-			page := url.Query().Get("page")
-			if page == "" {
+
```

Comment:
Hmmm... I'm not sure I'm happy with this section (especially regarding increasing the already-deep indentation levels).

Since `"cursor"` is the less-common and smaller case, let's please handle it first, and let's revert all the changes made to the handling of the `"page"` section, except maybe to pull out the shared `query` first... something like this:

```go
q := url.Query()
if cursor := q.Get("cursor"); cursor != "" {
...
}

page := q.Get("page")
if page == "" {
  continue
}
...
```

Reply:
Thanks! Addressed in f6d688a.